edge-apps/iframe/static/js/main.js

```diff
@@ -0,0 +1,20 @@
+/* global  screenly */
+
+function extractUrlFromIframeSetting (settingValue) {
+  if (settingValue.includes('<iframe')) {
+    const match = settingValue.match(/<iframe.*?src=["'](.*?)["']/)
+    if (match && match[1]) {
+      return match[1]
+    }
+  } else {
+    return settingValue
+  }
+  return null
+}
+document.addEventListener('DOMContentLoaded', function () {
+  const iframeSettingValue = screenly.settings.iframe || 'iframe URL not set'
+  const iframeUrl = extractUrlFromIframeSetting(iframeSettingValue)
+  const iframeElement = document.getElementById('iframe')
+  iframeElement.src = iframeUrl
+  iframeElement.innerText = iframeUrl
```

Comment:
Any particular reason for setting innerText to iframe URL here?

Reply:
Line number 18 might not be required. Looks good otherwise.

Reply:
@iamplugged You mean 19th right?
` iframeElement.innerText = iframeUrl`

Reply:
@iamplugged Done.